[Feature #112505171] Implement Creating Meetups Faqs

andela-aonyango opened the pull request:
The meetup creation page "/meetup/new" (also accessible form the Host a Meetup link in the Meetups dropdowns) has been created, as well as the meetup FAQ page "/meetup/faq".

Currently, there is no notification to even let the user know that a meetup has just been created successfully.

Users can create a meetup but the feature where the email is sent to the site administrator has yet to be implemented.

Users should also get a confirmation mail but that would be implemented in the next PR.
All the developers working on this project are expected to be administrators.

Commit:
(Feature #111860601) Create index view.

Commit:
(Feature #111860601) Change the URL of Meetup creation page to /meetup/new

Commit:
(Feature #111860601) Add DateTimePicker library.

Commit:
Merge branch 'staging' of https://github.com/andela/pibbble into fix/profile-placeholders

Commit:
(Fix) Remove search bug.

Commit:
(Feature #111860601) Create Meetups migration file.

Commit:
(Feature #111860601) Make Meetup creation route protected.

Commit:
(Feature #111860601) Add Meetups tests.

Commit:
(Fix) Solve houndci issue.

andela-aonyango committed:
[Feature ##112505171] Implement Creating Meetups Faqs

Added the meetups FAQ page. For now, it's not updated using
a database. That feature will be added once roles and a UI
for adding FAQs is added.

andela-aonyango committed:
[Feature #112505171] Implement Creating Meetups Faqs

app/Http/routes.php
Added a route to handle requests to the FAQ page

app/Http/Controllers/MeetupController.php
Added a function to return the FAQ view on request

resources/views/meetups/index.blade.php
Added the link to the FAQ page

andela-aonyango committed:
[Feature #112505171] Implement Creating Meetups Faqs

Added styling to match the meetups index page.

andela-aonyango committed:
Merge branch 'staging' of https://github.com/andela/pibbble into staging

unicodeveloper commented:
The `our FAQ` link should be more significant. You can do that by increasing the font size and changing the color to make it to a significant link.

<img width="595" alt="screen shot 2016-02-09 at 3 06 09 pm" src="https://cloud.githubusercontent.com/assets/2946769/12918289/b4627524-cf3e-11e5-941c-304973092a4e.png">

unicodeveloper commented:
Right now, a user can become a zombie/Michael Jackson from the past because a meetup can be created from the past :smile: 
<img width="661" alt="screen shot 2016-02-09 at 3 12 12 pm" src="https://cloud.githubusercontent.com/assets/2946769/12918454/889fcc6a-cf3f-11e5-932e-5a188e03d83d.png">

Please validate and fix that. :open_hands:

unicodeveloper commented on public/js/jquery.datetimepicker.full.min.js:
```diff
@@ -0,0 +1,2 @@
+var DateFormatter;!function(){"use strict";var e,t,a,n,r,o;r=864e5,o=3600,e=function(e,t){return"string"==typeof e&&"string"==typeof t&&e.toLowerCase()===t.toLowerCase()},t=function(e,a,n){var r=n||"0",o=e.toString();return o.length<a?t(r+o,a):o},a=function(e){var t,n;for(e=e||{},t=1;t<arguments.length;t++)if(n=arguments[t])for(var r in n)n.hasOwnProperty(r)&&("object"==typeof n[r]?a(e[r],n[r]):e ...
```
Ignore this, since it is a minified file.

andela-aonyango committed:
[Feature #112505171] Send email to meetup creator and admins

Added a method to send a confirmation email to the meetup creator
and the site administrators.

andela-aonyango committed:
[Feature #112505171] Refactor meetup index.blade and master.blade

Removed jquery datetime script reference from the main page to just
the meetup page, where it is used, instead of loading it on every
request to a Pibbble page.

Also removed unused class attributes from the meetup index.blade.php.

andela-aonyango committed:
[Feature #112505171] Validate input date from datetime picker

Added a Javascript function to ensure that dates in the past cannot
be provided as input.

Also changed the format of the date so that the month and day are
unambiguous.

andela-aonyango committed:
[Feature #112505171] Create a meetup-confirmation email template

Created an email template to be sent to meetup creators and site
administrators on successful creation of meetup.

andela-aonyango committed:
Applied fixes from StyleCI

andela-aonyango committed:
Merge pull request #319 from andela/analysis-qgORZ1

Applied fixes from StyleCI

andela-aonyango committed:
[Feature #112505171] Fix - Send email to meetup creator and admins

Added ability to send email to all admins and the person creating
the meetup as well.
